scout/test/dao/space_dao.py

```diff
@@ -213,6 +214,41 @@ def test_group_by_building(self):
                          "Odegaard Undergraduate Library")
         self.assertEqual(len(grouped_spots[1]), 2)
 
+    def test_organize_hours(self):
+        """
+        Tests the organize_hours function. Includes cases for overnight
+        and extremely long time periods.
+        """
+        spot = Spot()
+        time = datetime.time
```

Comment:
Recommend avoiding doing this simply because of how the datetime and time modules are organized: you've got time, datetime.time, and datetime.datetime.time that a developer who comes after you may want to import and utilize someday.